x/wasm/keeper/querier.go

```diff
@@ -311,3 +311,48 @@ func (q GrpcQuerier) PinnedCodes(c context.Context, req *types.QueryPinnedCodesR
 		Pagination: pageRes,
 	}, nil
 }
+
+func (q GrpcQuerier) InactiveContracts(c context.Context, req *lbmtypes.QueryInactiveContractsRequest) (*lbmtypes.QueryInactiveContractsResponse, error) {
```

Comment:
I think `ListInactiveContracts` or `GetInactiveContracts` is better name.

Reply:
But the name style of wasm did not use verbs. So I followed this rule.

https://github.com/line/lbm-sdk/blob/ded22b36a37906f250eb71db52bfe5b6b004aaf4/proto/cosmwasm/wasm/v1/query.proto#L14-L52